pkg/nettest/nettest.go

```diff
@@ -76,10 +76,15 @@ func Retryable(err error) bool {
 
 // RunWithNetRetry runs the given function and retries in case of network errors (see Retryable).
 func RunWithNetRetry(t *testing.T, fn func() error) error {
-	return retry.Do(func() error {
-		if err := fn(); err != nil && Retryable(err) {
+	var err error
+	// Do cannot return a non-nil error here.
```

Comment:
Ah, yeah that's true, because it will retry forever until the func returns nil, looks like we should also set a `maxAttempts` even if quite high?

EDIT: I see that it is only called from tests, so I guess at some point the test timeout would come into play, so probably not a big deal.

Reply:
@mna Yes, not a big deal. My problem with the function was that it always returned a nil error even thought the target-function returned an error.